app/assets/javascripts/iebacklink.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function() {
+    // =====================================================
+    // Back link mimics browser back functionality
+    // =====================================================
+    // store referrer value to cater for IE - https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/10474810/  */
+    var docReferrer = document.referrer
+    // prevent resubmit warning
+    if (window.history && window.history.replaceState && typeof window.history.replaceState === 'function') {
+        window.history.replaceState(null, null, window.location.href);
+    }
+    $('#back-link').on('click', function(e){
+        e.preventDefault();
```

Comment:
I may need to test this myself to be sure - but the ReturnToTop component should have the href `#main-content` instead of just `#` - as we want it to act similarly to the skip link and bring them to the top of <main>

This means the JS here should be amended to add the if statement so the back link doesn't break for IE after using the return to top link.
```
$('#back-link').on('click', function(e){
    e.preventDefault();
    if (window.location.href.indexOf("main-content") !== -1) {
        window.history.go(-2);
    } else {
        window.history.back();
    }
  })
```

Reply:
I've changed this now